modules/server/package.json

```diff
@@ -18,33 +18,33 @@
     "@ephox/bedrock-client": "^9.6.1",
     "@ephox/bedrock-common": "^9.3.2",
     "@ephox/bedrock-runner": "^9.6.1",
-    "async": "^1.5.2",
+    "async": "^3.2.0",
     "command-line-args": "^5.0.0",
-    "command-line-usage": "^5.0.1",
+    "command-line-usage": "^6.1.0",
     "core-js": "^3.6.4",
     "core-js-bundle": "^3.6.4",
     "cross-spawn": "^7.0.0",
     "etag": "^1.8.1",
     "finalhandler": "^1.1.2",
-    "fork-ts-checker-webpack-plugin": "~1.0.4",
+    "fork-ts-checker-webpack-plugin": "~5.0.5",
     "fresh": "^0.5.2",
     "grunt": "^1.0.4",
     "istanbul-instrumenter-loader": "^3.0.1",
     "jquery": "^3.4.1",
-    "load-grunt-tasks": "^3.5.0",
-    "mkdirp": "^0.5.1",
+    "load-grunt-tasks": "^5.1.0",
+    "mkdirp": "^1.0.4",
     "node-glob": "^1.2.0",
     "portfinder": "^1.0.25",
-    "raw-loader": "^0.5.1",
+    "raw-loader": "^4.0.1",
     "recursive-readdir-sync": "^1.0.6",
-    "rollup": "^1.23.1",
-    "rollup-plugin-typescript2": "^0.24.0",
+    "rollup": "^2.18.1",
+    "rollup-plugin-typescript2": "^0.27.1",
     "serve-static": "^1.10.2",
-    "source-map-loader": "^0.2.3",
-    "ts-loader": "^5.3.3",
+    "source-map-loader": "^1.0.0",
+    "ts-loader": "^7.0.5",
```

Comment:
This can't be updated yet... last time I tried it broke a lot of things, so we need to be very careful about updating a number of these packages. This really needs an investigation/testing task assigned to it as well, given the potential it has for breaking things.